scripts/run-integration-tests.sh

```diff
@@ -107,7 +106,7 @@ get_exasol_certificate_fingerprint() {
 
     fingerprint=$(echo "$certificate" \
                 | openssl x509 -fingerprint -sha256 -noout -in /dev/stdin \
-                | sed 's/SHA256 Fingerprint=//' \
+                | sed 's/SHA256 Fingerprint=//i' \
```

Comment:
Github action runner was upgraded. The new `openssl` outputs `sha256` in lower case.